addons/website_livechat/__manifest__.py

```diff
@@ -30,18 +30,18 @@
             "website_livechat/static/src/**/common/**/*",
         ],
         'website.assets_wysiwyg': [
-            'website_livechat/static/src/scss/**/*',
+            "website/static/src/**/common/**/*",
```

Comment:
Similar remark as the original PR, I strongly believes modules should define their own files.

Fix should be in `website`, it's irrelevant to `website_livechat`.